curriculum/challenges/spanish/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.spanish.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: 587d7fb1367417b2b2512bf4
 title: Chain Middleware to Create a Time Server
-localeTitle: Cadena de middleware para crear un servidor de tiempo
+localeTitle: Encadena un middleware para crear un servidor de tiempo
 challengeType: 2
 ---
 
 ## Description
 <section id='description'> 
-Middleware puede montarse en una ruta específica usando <code>app.METHOD(path, middlewareFunction)</code> . Middleware también puede ser encadenado dentro de la definición de ruta. 
+un Middleware puede montarse en una ruta específica usando <code>app.METHOD(path, middlewareFunction)</code> . Los Middlewares también pueden ser encadenados dentro de la definición de ruta. 
```

Comment:
```suggestion
Un Middleware puede montarse en una ruta específica usando <code>app.METHOD(path, middlewareFunction)</code> . Los Middlewares también pueden ser encadenados dentro de la definición de ruta. 
```
But more importantly, should middleware really be capitalized? Now it's not consistent in the challenge any more.